shells/lib/utils.js

```diff
@@ -56,8 +56,9 @@ const resolve = async (arc, recipe) =>{
 };
 
 const spawn = async ({id, serialization, context, composer, storage}) => {
+  const arcId = IdGenerator.newSession().createArcId(id);
```

Comment:
_new_ session, but _create_ arcId. Is there a semantic difference you're aiming to convey?

Using these 2 verbs reads slightly weird to me. If there is no difference maybe go for `newArcId(...)` for brevity and consistency?

`IdGenerator.newSession().newArcId(id);`

Reply:
I like it, done!